django_cyverse_auth/authBackends.py

```diff
@@ -352,8 +352,8 @@ def keystone_validate_token(self, auth_url, username, token, project_name, proje
             project_name=project_name,
             project_domain_id=project_domain)
         try:
-            self._keystone_auth_to_token(token_auth, username, project_name)
-            return self._update_token(auth_url, username, token, request)
+            token, email = self._keystone_auth_to_token(token_auth, username, project_name)
```

Comment:
It would be more generally useful to return `session, token` (note the arg-swap) here.
```
session, token = self._keystone_auth_to_token(...)
email = self._lookup_email(session)
```